src/components/dashboard/dashboard.component.less

```
@@ -11,8 +11,8 @@ ux-dashboard {
 
         .position-indicator {
             position: absolute;
-            border: 1px solid @grey6;
-            background-color: @grey7;
+            outline: 1px dashed #00a7a2;
```

```suggestion
            outline: 1px dashed @primary;
```
Think this can use the primary variable